clusterloader2/testing/experimental/storage/pod-startup/ephemeral-volumes/1_node/config.yaml

```diff
@@ -0,0 +1,95 @@
+# ASSUMPTIONS:
+# - this test cases should be run on only 1 node
+
+#Constants
+{{$TOTAL_PODS := DefaultParam .TOTAL_PODS 1}}
+{{$POD_TEMPLATE_PATH := DefaultParam .POD_TEMPLATE_PATH "pod_with_EmptyDirs.yaml"}}
+{{$LOAD_TEST_THROUGHPUT := DefaultParam .LOAD_TEST_THROUGHPUT 10}}
+
+{{$VOLUMES_PER_POD := DefaultParam .VOLUMES_PER_POD 4}}
+{{$VOLUME_TEMPLATE_PATH := .VOLUME_TEMPLATE_PATH}}
+{{$PROVISION_VOLUME := DefaultParam .PROVISION_VOLUME false}}
+
+{{$APP_NAME := "pod-load"}}
+{{$GROUP := "pod-with-ephemeral-volume-startup-latency"}}
+
+#Variables
+# used stateless pod SLO for initial run
+{{$podStartupTimeout := 5}}
+{{$totalVolumes := MultiplyInt $TOTAL_PODS $VOLUMES_PER_POD}}
+
+name: pod-with-ephemeral-volume-startup-latency
+automanagedNamespaces: 1
+tuningSets:
+- name: UniformQPS
+  qpsLoad:
+    qps: {{$LOAD_TEST_THROUGHPUT}}
+steps:
+# Start measurements
+- measurements:
+  - Identifier: APIResponsiveness
+    Method: APIResponsiveness
+    Params:
+      action: reset
+- measurements:
+  - Identifier: PodWithMultiVolumeStartupLatency
+    Method: PodStartupLatency
+    Params:
+      action: start
+      labelSelector: group = {{$GROUP}}
+      threshold: {{$podStartupTimeout}}s
+{{ if $PROVISION_VOLUME }}
+# Provision volumes
+- phases:
+  - namespaceRange:
+      min: 1
+      max: 1
+    replicasPerNamespace: {{$totalVolumes}}
+    tuningSet: UniformQPS
+    objectBundle:
+    - basename: vol-{{$APP_NAME}}
+      objectTemplatePath: {{$VOLUME_TEMPLATE_PATH}}
+{{ end }}
+# Create pods
+- phases:
+  - namespaceRange:
+      min: 1
+      max: 1
+    replicasPerNamespace: {{$TOTAL_PODS}}
+    tuningSet: UniformQPS
+    objectBundle:
+    - basename: {{$APP_NAME}}
+      objectTemplatePath: {{$POD_TEMPLATE_PATH}}
+      templateFillMap:
+        Group: {{$GROUP}}
+        VolumesPerPod: {{$VOLUMES_PER_POD}}
+        AppName: {{$APP_NAME}}
+- measurements:
+  - Identifier: WaitForRunningPodsWithStorage
+    Method: WaitForRunningPods
+    Params:
+      desiredPodCount: {{$TOTAL_PODS}}
+      labelSelector: group = {{$GROUP}}
+      # TODO decide this after test roll-out phase
+      timeout: 15m
+- measurements:
```

Comment:
Similarly as for starting measurement, can you also move it to the bottom?

[As I mentioned somewhere else, medium term, we will simply bake all the necessary thresholds into the measurement itself and we will bundle all SLO-related ones together to limit boilerplate. It will just be simpler to do then :)]

Reply:
ok., I think deleting pods before the gather action of PodStartupLatency measurement has no effect?

Reply:
PodStartupLatency measurement can only change if when pods are being created - so yes, that will have no effect.